cartridges/int_algolia_sfra/cartridge/static/default/js/algolia/autocomplete-config.js

```diff
@@ -62,11 +128,44 @@ function getSourcesArray(config) {
 
     var sourcesArray = [];
 
+    sourcesArray.push({
+        sourceId: 'trendingProducts',
```

Comment:
When search results start to appear, this source shouldn't stay at the top of the results. It should either be added last, or displayed on the side of the results.